schema/dalmatinerdb.schema

```diff
@@ -10,6 +10,12 @@
  [{default, 100},
   {datatype, integer}]}.
 
+%% @doc Number of allowed concurrent tcp connections.
+{mapping, "tcp_max_connections", "dalmatiner_db.tcp_max_connections",
+ [{default, 1024},
+  {datatype, integer},
+  hidden]}.
```

Comment:
I would go w/o the hidden here, this seems to be a nice thing to have in general, no reason to hide it from users.